packages/discovery-provider/src/challenges/listen_streak_challenge.py

```diff
@@ -33,6 +36,45 @@ def get_listen_streak_override(session: Session, user_id: int) -> Optional[int]:
 class ChallengeListenStreakUpdater(ChallengeUpdater):
     """Listening streak challenge"""
 
+    def _check_recent_listen_streak(
```

Comment:
i would rename this method `_get_current_listen_streak` which returns none if there is no current streak